index.js

```diff
@@ -40,7 +40,6 @@ const styles = StyleSheet.create({
     height: '100%',
     alignItems: 'center',
     justifyContent: 'space-around',
-    zIndex: -1
```

Comment:
This `zIndex` is added as part of `edgeRadius` property in #24 

We may have to figure a better way to work with `edgeRadius`.